tensorboard/components/tf_dashboard_common/tf-multi-checkbox.ts

```diff
@@ -21,6 +21,12 @@ Polymer({
       type: Array,
       value: () => [],
     },  // All the values of checkbox
+    colors: {
```

Comment:
Perhaps "colorScale" or "coloring" might still be a better name for this property?  The name "colors" sounds to me more like just a plain array of color values.  Ditto on tf-runs-selector.